flink-state-backends/flink-statebackend-rocksdb/src/main/java/org/apache/flink/contrib/streaming/state/RocksDBOptions.java

```diff
@@ -62,6 +63,16 @@ public class RocksDBOptions {
                             .withDescription(
                                     "This determines the factory for timer service state implementation.");
 
+    @Documentation.Section(Documentation.Sections.STATE_BACKEND_ROCKSDB)
+    public static final ConfigOption<Integer> ROCKSDB_TIMER_SERVICE_FACTORY_CACHE_SIZE =
+            ConfigOptions.key("state.backend.rocksdb.timer-service.cache-size")
+                    .intType()
+                    .defaultValue(DEFAULT_CACHES_SIZE)
+                    .withDescription(
+                            String.format(
+                                    "The cache size for rocksdb timer service factory. This option only has an effect when '%s' is configured to '%s'",
```

Comment:
Also add a simple description about benefits and costs about increasing/decreasing the value ? 
It could help users to know how to adjust the value.

Reply:
Good idea. I have add a simple description about this.